.azure-pipelines/templates/platform_ci_pipeline.yml

```diff
@@ -138,6 +140,8 @@ stages:
     - stage: 'Approve_Model'
       dependsOn: 
       - execute_training_job
+      - Deploy_Batch
```

Comment:
in fact, failed deployment doesn't mean that we cannot approve the model. So, I would recommend to remove these two lines.

Reply:
Should approval only occur after both deployments are successful? Because the review was popping after training completed.